### PR TITLE
Extended Stillsuit support

### PR DESCRIPTION
**Pull request by gausie**
Outstanding questions:

- What do we do with the current Buzzed on Distillate effect (easy enough to parse modifiers and store in a pref)
- What do we do with the next Buzzed on Distillate effect (at the moment I am parsing the modifiers but not sure what to do with them)

EDIT: I've tied this up to work as is, better support can (and probably will!) come down the line

### CODE REVIEW

File: src/net/sourceforge/kolmafia/request/GenericRequest.java [hunk 1]
```diff
@@ -1393,15 +1393,7 @@ public void externalExecute() {
...
-    return requestURL == null
-        ||
-        // Disallow mall searches
-        requestURL.contains("mall.php")
-        || requestURL.contains("manageprices.php")
-        || requestURL.contains("backoffice.php")
-        ||
-        // Disallow anything to do with chat
-        request.isChatRequest;
+    return false;
```
Comment (midgleyc) [inline src/net/sourceforge/kolmafia/request/GenericRequest.java:1396]: is this change related, or has it snuck in?

Comment (gausie) [reply src/net/sourceforge/kolmafia/request/GenericRequest.java:1396]: One day I'll stop accidentally adding this... test... code

File: src/net/sourceforge/kolmafia/session/StillSuitManager.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    var familiar =
+        KoLCharacter.familiars.stream()
+            .filter(f -> f.getItem() != null && f.getItem().getItemId() == ItemPool.STILLSUIT)
+            .findFirst()
+            .orElse(null);
+
+    int drams = 0;
+    if (familiar == KoLCharacter.getFamiliar()) {
+      drams = 2;
```
Comment (midgleyc) [inline src/net/sourceforge/kolmafia/session/StillSuitManager.java:32]:
```suggestion
      drams = 3;
```

Comment (midgleyc) [reply src/net/sourceforge/kolmafia/session/StillSuitManager.java:32]: unless I am wrong about what this is, equipped familiars give 3 drams of sweat?

Comment (gausie) [reply src/net/sourceforge/kolmafia/session/StillSuitManager.java:32]: You're right! I forgot

**Review by midgleyc — APPROVED** [review]
